### PR TITLE
fix: production issue

### PR DESCRIPTION
**Pull request by gengjiawen**
close https://github.com/blitz-js/blitz/issues/4303
close https://github.com/blitz-js/blitz/issues/4310

### CODE REVIEW

**Comment by cfragkos** [comment]
sadly this will cause next 14 on vercel to fail #4309 but it does at least fix your problems

**Comment by gengjiawen** [comment]
> sadly this will cause next 14 on vercel to fail #4309 but it does at least fix your problems

I test locally this works for next 14. Not sure how it's on vercel

**Comment by gengjiawen** [comment]
@siddhsuresh Any ETA this can be merged ?

File: .changeset/soft-tables-ring.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@blitzjs/auth": patch
+---
+
+fix production issues
```
Comment (siddhsuresh) [inline .changeset/soft-tables-ring.md:5]:
```suggestion
Fix bundling issue that occurs in vercel due to the way imports were handled internally
```

**Comment by gengjiawen** [comment]
@siddhsuresh Can you draft a new release for this ? thx.

**Comment by siddhsuresh** [comment]
on it @gengjiawen. Looks like release CI is broken, will fix that, if not will manually release

**Pull request merged**